src/components/application_manager/test/commands/mobile/alert_request_test.cc

```diff
@@ -105,27 +197,24 @@ TEST_F(AlertRequestTest, OnTimeOut_SUCCESS) {
 }
 
 TEST_F(AlertRequestTest, Run_ApplicationIsNotRegistered_UNSUCCESS) {
-  CommandPtr command(CreateCommand<AlertRequest>());
+  CommandPtr command(CreateCommand<AlertRequest>(msg_));
```

Comment:
@okozlovlux Move variables to the place before their first usage. In all test cases please.